TaffoInitializer/TaffoInitializerPass.cpp

```diff
@@ -509,7 +513,14 @@ Function* TaffoInitializer::createFunctionAndQueue(llvm::CallSite *call, ConvQue
   LLVM_DEBUG(dbgs() << "  callsite instr " << *call->getInstruction() << " [" << call->getInstruction()->getFunction()->getName() << "]\n");
   for (int i=0; oldArgumentI != oldF->arg_end() ; oldArgumentI++, newArgumentI++, i++) {
     Value *callOperand = call->getInstruction()->getOperand(i);
-    Value *allocaOfArgument = newArgumentI->user_begin()->getOperand(1);
+
+    auto user_begin = newArgumentI->user_begin();
+    if (user_begin == newArgumentI->user_end()) {
+      LLVM_DEBUG(dbgs() << "  Arg nr. " << i << " skipped, callOperand has no valueInfo\n");
+      continue;
+    }
```

Comment:
Heh, call operands without valueInfo shouldn't happen. I'm letting this one pass, though, because valueInfo handling is a huge and unsalvageable rat's nest

Reply:
I dived deep some months ago in this issue and I honestly forgot which case fired an issue there 😭 - now that we have some test cases I'm going to re-run them for curiosity. I'll try to see and  add a comment explaining how/why it happens (if it's still happening 🙏 )

Reply:
I checked a bit better again and the problem is in the OpenMP internal arguments (`loc` and `argc`) that have no uses, therefore the subsequent `getOperand(1)` on `user_begin` fails . 

I fixed the debug message and kept the check